### PR TITLE
Konnectivity example refresh.

### PR DESCRIPTION
**Pull request by jkh52**
- Delete UDS file if exists (common pain point).
- Make server and agent images consistent (and include latest memory leak fixes).

Relevant discussion: https://github.com/kubernetes-sigs/apiserver-network-proxy/pull/476

### CODE REVIEW

**Comment by windsonsea** [comment]
Hi @jkh52 , you only need to include one language (such as en) in a PR, and leave other languages to localization teams.

**Comment by jkh52** [comment]
> Hi @jkh52 , you only need to include one language (such as en) in a PR, and leave other languages to localization teams.

Updated to `en` only.

File: content/en/examples/admin/konnectivity/konnectivity-server.yaml [hunk 1]
```diff
@@ -8,12 +8,13 @@ spec:
...
   containers:
   - name: konnectivity-server-container
-    image: registry.k8s.io/kas-network-proxy/proxy-server:v0.0.32
+    image: registry.k8s.io/kas-network-proxy/proxy-server:v0.0.37
     command: ["/proxy-server"]
     args: [
             "--logtostderr=true",
             # This needs to be consistent with the value set in egressSelectorConfiguration.
             "--uds-name=/etc/kubernetes/konnectivity-server/konnectivity-server.socket",
+            "--delete-existing-uds-file=true",
```
Comment (tallclair) [inline content/en/examples/admin/konnectivity/konnectivity-server.yaml:17]: nit: do you need `=true`, or can the flag just be `--delete-existing-uds-file`?

**Review by tallclair — COMMENTED** [review]
/lgtm

File: content/en/examples/admin/konnectivity/konnectivity-server.yaml (hunk 1 above)
Comment (jkh52) [reply content/en/examples/admin/konnectivity/konnectivity-server.yaml]: I verified that the `=true` is not needed.

**Comment by jkh52** [comment]
Friendly ping.

@windsonsea

**Comment by jkh52** [comment]
/assign tengqm

(Choosing a recent active sig-docs-en-reviews member; let me know if there's a different process to request approval)

**Comment by tengqm** [comment]
/approve

Thanks.

**Pull request merged**